.openpublishing.redirection.json

```diff
@@ -1022,6 +1022,14 @@
         {
             "source_path": "docs/csharp/programming-guide/strings/how-to-convert-rtf-to-plain-text.md",
             "redirect_url": "https://msdn.microsoft.com/en-us/library/cc488002(v=vs.140).aspx"
+        },
+        {
+            "source_path": "docs/csharp/parallel.md",
+            "redirect_url": "/dotnet/csharp/programming-guide/threading/index.md"
+        },
+        {
+            "source_path": "docs/csharp/methods-lambda-expressions.md",
+            "redirect_url": "/dotnet/csharp/methods.md"
```

Comment:
I don't think that methods.md is the appropriate redirect; that mentions lambda expressions only once (they're used for async methods). Better would be programming-guide/statements-expressions-operators/lambda-expressions.md.